src/app/common/services/form/form.service.ts

```diff
@@ -30,8 +30,8 @@ export const realEstateForm = new FormGroup({
   price: new FormControl('', [Validators.required])
 });
 
-  export const agentForm = new FormGroup({
-    name: new FormControl('', [Validators.required]),
-    email: new FormControl('', [Validators.required, Validators.email]),
-    phoneNumber: new FormControl('', [Validators.required]),
-  });
+export const agentForm = new FormGroup({
+  name: new FormControl('', [Validators.required, Validators.minLength(5), Validators.maxLength(100)]),
+  email: new FormControl('', [Validators.required, Validators.email, Validators.minLength(9), Validators.maxLength(13)]),
+  phoneNumber: new FormControl('', [Validators.required, Validators.minLength(8), Validators.maxLength(150)]),
```

Comment:
I think it would be better for this field to not accept letters

Reply:
the phone number should be "+351987654321" so it does need the letters